ert/storage/_storage.py

```diff
@@ -36,6 +29,11 @@
 _PARAMETER_RECORD_SEPARATOR = "."
 
 
+class ContentTypeEnum(enum.Enum):
```

Comment:
Given the usage of this class, I think it would be better to just drop the Enum inheritance for the class. 

```python 
class ContentTypeEnum:
    OCTET_STREAM = "application/octet-stream"
    CSV = "text/csv"
```

Reply:
replaced with namedtuple